lib/impl/skia/image.cpp

```diff
@@ -16,10 +16,24 @@
 
 #include "opaque.hpp"
 #include <stdexcept>
+#include <map>
 #include <string>
+#include <utility> // std::pair
+#include <iostream>
+
+using std::map;
+using std::pair;
 
 namespace cycfi::artist
 {
+   static const map<img_fmt, pair<SkAlphaType, SkColorType>> _img_fmt_map_to_api_type = {
```

Comment:
I'd probably just use a function and a switch/cases here. More efficient. No additional runtime usage. And more importantly, no globals.

Reply:
Good point! Will update.

Reply:
Updated to use fn.